2_base_and_sensor.ipynb

```diff
@@ -73,11 +114,55 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "**For advanced users** To move the move base, we use the `move_base` function defined in the `utils` package. The code is as follows:\n",
+    "\n",
+    "```python\n",
+    "from geometry_msgs.msg import Twist\n",
+    "\n",
+    "# Create a velocity command publisher\n",
+    "base_vel_pub = rospy.Publisher ('/hsrb/command_velocity', Twist, queue_size=1)\n",
+    "\n",
+    "def move_base_vel (vx, vy, vw):\n",
+    "    u \"\"\"Function that controls the velocity of the move base\n",
+    "\n",
+    "    argument:\n",
+    "        vx (float): Velocity command value in the straight direction [m/s] (forward is positive, reverse is negative)\n",
+    "        vy (float): Velocity command value in the lateral direction [m/s] (left is positive, right is negative)\n",
+    "        vw (float): Velocity command value in the rotation direction [deg/s] (anticlockwise rotation is positive, clockwise rotation is negative)\n",
+    "\n",
+    "    \"\"\"\n",
+    "\n",
+    "    # Set velocity command values\n",
+    "    twist = Twist ()\n",
+    "    twist.linear.x = vx\n",
+    "    twist.linear.y = vy\n",
+    "    twist.angular.z = vw / 180.0 * math.pi # Convert from \"degree\" to \"radian\"\n",
+    "    base_vel_pub.publish (twist) # Publish velocity command\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "まず、台車を前進させてみましょう。直進させたい場合は以下のようにの速度司令を入力します。ロボットが少し前進すると思います。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "First, let's move the move base forward. If you want to go straight, enter the velocity command as below. The robot should move forward a little:"
```

Comment:
If there is one velocity command, then use this:

    "First, let's move the moving base forward. If you want to go straight, enter the velocity command as shown below. The robot should move forward a little:"

If there are multiple velocity commands, then use this:

    "First, let's move the moving base forward. If you want to go straight, enter the velocity commands as shown below. The robot should move forward a little:"